### PR TITLE
Fixes #1, #2, #44, #46 - Reworking CLI tool

### LINKED ISSUE #1: Add CLI tool for easy installation of configs
**Is your feature request related to a problem? Please describe.**
Manually adding the configs are slow.

**Describe the solution you'd like**
Using `npx` tool to run CLI tool without install.

**Describe alternatives you've considered**
None

**Additional context**
Nonee.

### PR DESCRIPTION
**Pull request by simmo**
**What does this PR cover?**
Refactor + feature

**Breaking change?**
No

**Are all the tests are passing?**
Yes

**Other information:**
Initial rework of Javis cli tool - also adds file support for .editorConfig file.

### CODE REVIEW

**Comment by simmo** [comment]
The CLI doesn't have any at the moment... packages are 100% coverage though.
Ideally after input on if the architecture/code is ok.

**Comment by sburrows-zone** [comment]
not looked at each file in detail, but it is much clearer. Editor config looked fine to me

File: packages/jarvis/src/install.js [hunk 1]
```diff
@@ -1,114 +1,52 @@
...
+const { log } = require('./util/console')
+const prompt = require('./util/prompt')
+const resolveSerial = require('./util/resolveSerial')
+
+const filesTask = require('./tasks/files')
+const dependenciesTask = require('./tasks/dependencies')
+const configChangesTask = require('./tasks/configChanges')
+
+// Export the addItem functions from the tasks
+exports.addConfigChange = configChangesTask.addItem
```
Comment (arashkiani) [inline packages/jarvis/src/install.js]: Just wondering why we're using require & exports over import & export?

Comment (simmo) [reply packages/jarvis/src/install.js]: Last time I check ECMAScript modules are experimental in Node so you have to use Babel or similar to transpile to common JS. This project isn't yet using Babel for Node.

File: packages/zoneify/src/util/logger.test.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
+/* eslint-disable no-console */
```
Comment (arashkiani) [inline packages/zoneify/src/util/logger.test.js]: What do you think about adding something like 👇 in package.json rules, sometimes in the future?

```
"eslint-disable no-console": [
        "error",
        {
          "devDependencies": [
            "**/*.test.js",
          ]
        }
      ]
```

File: packages/zoneify/src/util/logger.test.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+const {
+  formatter, log, levels, spinner,
+} = require('./logger')
+
+global.console = {
```
Comment (arashkiani) [inline packages/zoneify/src/util/logger.test.js]: 👍

**Comment by simmo** [comment]
Looks like I have two test failing, but not sure why as they pass on local!!! Help!

I'll investigate.

Worth saying I've pushed at alpha release to NPM so you should be able to run `npx zoneify` in your terminal.

File: packages/zoneify/src/util/logger.test.js (hunk 2 above)
Comment (simmo) [reply packages/zoneify/src/util/logger.test.js]: Yeah not against it. Given I think there is only one place current I'm just using a one off exception.

**Comment by simmo** [comment]
The tests are fixed now so going to merge.

**Pull request merged**